test/simulation/tests/test_node.py

```diff
@@ -163,9 +171,17 @@ def test_add_peer(nodes):
     assert len(nodes[0].list_peer()) == 0
     assert len(nodes[1].list_peer()) == 0
     nodes[0].add_peer(nodes[1])
+    nodes[0].wait_for_log(
+        map_log_enum_pattern[RPCLogs.LOG_ADD_PEER_FMT],
+        0,
+    )
+    nodes[0].wait_for_log(
+        map_log_enum_pattern[RPCLogs.LOG_ADD_PEER_FINISHED],
```

Comment:
If we wait for `*_FINISHED` logs, what's the reason to wait for `*_FMT` prior to `*_FINISHED`?

Reply:
The difference between `*_FMT` and `*_FINISHED` is, there are arguments in `*_FMT`, while `*_FINISHED` not. I think `*_FMT` will be useful in this case, when we want to make sure an event with specific arguments has finished.